etc/install_package_dependencies.sh

```diff
@@ -1,30 +1,28 @@
-# This script can be used to install all the required package dependencies on a clean system
-
+#This script can be used to install all the required package dependencies on a clean system
+ 
 PROJECT_PATH=$(pwd)
 
 # Install Python-PIP
 sudo apt-get update
-sudo apt-get install -y python-pip
+sudo apt-get install -y python3-pip
+
+# Install virtualenv, create a virtual environment and activate it
+sudo -H pip3 install virtualenv
+virtualenv env
+source env/bin/activate
 
 
 # Install Graph-Tool
 sudo apt-get install -y expat
 sudo apt-get install -y libsparsehash-dev
-sudo apt-get install -y gtk+3
+sudo apt-get install -y gtk 3
 sudo apt-get install -y libboost-all-dev
 sudo apt-get install -y build-essential
 sudo apt-get install -y libcairo2-dev
-sudo apt-get install -y python3-pip
-sudo apt-get install -y python3-dev
-sudo apt-get install -y python3-matplotlib
 sudo apt-get install -y gfortran libopenblas-dev liblapack-dev
 sudo apt-get install -y libcgal-dev
-sudo apt-get install -y python3-numpy
-sudo apt-get install -y python2.7-config
-sudo apt-get install -y python3-cairo
-sudo apt-get install -y python3-scipy
 sudo apt-get install -y graphviz graphviz-dev libgraphviz-dev pkg-config
-sudo apt-get install -y python3-pygraphviz
```

Comment:
Is this a pure python library? If so, has this been moved to requirements.txt?

Reply:
Yes, it's present in requirements.txt